Automattic/wp-calypso
# Jetpack Cloud: Use calypso entrypoint

#### Changes proposed in this Pull Request
Remove the current jetpack cloud entry point. 

<img width="1116" alt="Screen Shot 2020-02-27 at 4 00 08 PM" src="https://user-images.githubusercontent.com/115071/75455991-4105c200-597a-11ea-8cd6-618027122a30.png">


Some regression in this PR. 
1. The design falls back to calypso. (no more blackmasterbar) since we are not including the styles anymore.
2. Doesn't limit the sites in jetpack cloud  to just jetpack sites. 

I think they should be addressed in a different PR.

#### Testing instructions
1. Start up calypso `npm start` 
2. notice that nothing seems to be broken and that we can navigate to all the sections as expected. 
3. Notice that you can't visit sections like `scan` and `backup`

Test cloud. 
1. Start up jetpack cloud `npm start start-jetpack-cloud` 
2. Notice the regressions but you should be able to visit all the other section as expected.
